data-otservbr-global/lib/core/storages.lua

```diff
@@ -2392,51 +2235,261 @@ Storage = {
+				Mission35 = 44238,
+				Mission36 = 44239,
+				Mission36a = 44240,
+				Mission37 = 44241,
+				Mission38 = 44242,
+				Mission38a = 44243,
+				Mission38b = 44244,
+				Mission38c = 44245,
+				Mission39 = 44246,
+				Mission40 = 44247,
+				Mission41 = 44248,
+				Mission42 = 44249,
+				Mission43 = 44250,
+				Mission44 = 44251,
+				Mission45 = 44252,
+				Mission46 = 44253,
+				Mission47 = 44254,
+				Mission48 = 44255,
+				Mission49 = 44256,
+				Mission50 = 44257,
+				Mission51 = 44258,
+				Mission52 = 44259,
+				Mission53 = 44260,
+				Mission54 = 44261,
+				Mission55 = 44262,
+				Mission56 = 44263,
+				Mission57 = 44264,
+				Mission58 = 44265,
+				Mission59 = 44266,
+				Mission60 = 44267,
+				Mission61 = 44268,
+				Mission62 = 44269,
+				Mission63 = 44270,
+				Mission64 = 44271,
+				Mission65 = 44272,
+				Mission66 = 44273,
+				Mission67 = 44274,
+				Mission68 = 44275,
+				Mission69 = 44276,
+				Mission70 = 44277,
+				Mission71 = 44278,
+				Mission72 = 44279,
+				Mission73 = 44280,
+				Mission74 = 44281,
+				Mission75 = 44282,
+				Mission76 = 44283,
+				Bookcase = 44285,
+				SmallDragonTears1 = 44286,
+				SmallDragonTears2 = 44287,
+				SmallDragonTears3 = 44288,
+			},
 			TheRepenters = {},
 		},
 		U10_20 = { -- update 10.20 - Reserved Storages 44301 - 44400
-			CaveExplorerOutfits = {},
-			SpikeTasks = {},
+			SpikeTaskQuest = {
+				QuestLine = 44301,
+				Gnomilly = 44302,
+				Gnombold = {
+					Points = 44303,
+					Nests = 44304,
+					Fertilise = 44305,
+					Kill = 44306,
+					Charges = 44307,
+				},
+				Gnomargery = {
+					Points = 44308,
+					Deliver = 44309,
+					Undercover = 44310,
+					Temperature = 44311,
+					Kill = 44312,
+				},
+				Constants = {
+					Spike_Fame_Points = 44313,
+					Spike_Upper_Pacifier_Main = 44314,
+					Spike_Upper_Pacifier_Daily = 44315,
+					Spike_Upper_Mound_Main = 44316,
+					Spike_Upper_Mound_Daily = 44317,
+					Spike_Upper_Track_Main = 44318,
+					Spike_Upper_Track_Daily = 44319,
+					Spike_Upper_Kill_Main = 44320,
+					Spike_Upper_Kill_Daily = 44321,
+					Spike_Middle_Charge_Main = 44322,
+					Spike_Middle_Charge_Daily = 44323,
+					Spike_Middle_Mushroom_Main = 44324,
+					Spike_Middle_Mushroom_Daily = 44325,
+					Spike_Middle_Nest_Main = 44326,
+					Spike_Middle_Nest_Daily = 44327,
+					Spike_Middle_Kill_Main = 44328,
+					Spike_Middle_Kill_Daily = 44329,
+					Spike_Lower_Parcel_Main = 44330,
+					Spike_Lower_Parcel_Daily = 44331,
+					Spike_Lower_Undercover_Main = 44332,
+					Spike_Lower_Undercover_Daily = 44333,
+					Spike_Lower_Lava_Main = 44334,
+					Spike_Lower_Lava_Daily = 44335,
+					Spike_Lower_Kill_Main = 44336,
+					Spike_Lower_Kill_Daily = 44337,
+				},
+			},
 			SpiritGroundsMiniWorldChange = {},
 		},
 		U10_30 = { -- update 10.30 - Reserved Storages 44401 - 44500
 			DreamWardenOutfits = {},
-			Roshamuul = {},
+			RoshamuulQuest = {
+				Constants = {
+					Roshamuul_Mortar_Thrown = 44401,
```

Comment:
I think the naming standard should be maintained and I also believe that they are not constant.